README.md

```diff
@@ -161,3 +163,12 @@ uvicorn therapy.main:app --reload
 Next, view the OpenAPI docs on your local machine:
 
 http://127.0.0.1:8000/therapy
+
+
+### FAQ
+
+**A data import method raised a SourceFormatException instance. How do I proceed?**
```

Comment:
Is this a bioversions exception? Might be useful to specify which dependency would raise this

Reply:
ope that's from a different issue